orchagent/dash/dashtunnelorch.cpp

```diff
@@ -172,14 +187,17 @@ void DashTunnelOrch::doTask(ConsumerBase &consumer)
                 }
                 else
                 {
+                    result = DASH_RESULT_FAILURE;
                     it_prev++;
                 }
+                writeResultToDB(dash_tunnel_result_table_, tunnel_name, result);
```

Comment:
We should only be writing the result here if the tunnel only has one endpoint, if it has multiple then we need to wait until after `addTunnelMembersPost`